p2p/receiver/handler_integration_test.go

```diff
@@ -146,7 +153,7 @@ func TestHandler_HandleInterceptorReqSignature(t *testing.T) {
 	assert.NotNil(t, resp.Signature.Signature, "must be non nil")
 	sig := resp.Signature
 	doc := dm.Document
-	assert.True(t, ed25519.Verify(sig.PublicKey, doc.SigningRoot, sig.Signature), "signature must be valid")
+	assert.True(t, secp256k1.VerifySignatureWithAddress(common.BytesToAddress(sig.PublicKey).String(), hexutil.Encode(sig.Signature), doc.SigningRoot), "signature must be valid")
```

Comment:
I think it's much better to call secp directly like it is done here.

Reply:
I would like to keep some wrapping for signing and verification in `crypto` package so that we don't endup having to refactor many places when changing the signature schemes

Reply:
I agree on the wrapper, I havent added yet but was on my mind as soon as I got confirmation that we didnt need any other type of signing for secp curves.